int32 length
int32 type 
int32 command
int32 reply
int32 unused

float32[6] jts

---

int32 bytes_sent
